pkgs/development/compilers/wla-dx/default.nix

```diff
@@ -0,0 +1,26 @@
+{stdenv, fetchFromGitHub, cmake}:
+
+stdenv.mkDerivation rec {
+  name = "wla-dx-${rev}";
+  rev = "8189fe8d5620584ea16563875ff3c5430527c86a";
```

Comment:
Is there a reason to use HEAD rather than v9.6? (apart from it being a bit old, that is)